src/test/java/org/openstreetmap/atlas/geography/boundary/DynamicGridIndexBuilderTest.java

```diff
@@ -0,0 +1,133 @@
+package org.openstreetmap.atlas.geography.boundary;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.geotools.measure.Longitude;
+import org.junit.Assert;
+import org.junit.Test;
+import org.openstreetmap.atlas.geography.Latitude;
+import org.openstreetmap.atlas.streaming.compression.Decompressor;
+import org.openstreetmap.atlas.streaming.resource.InputStreamResource;
+import org.openstreetmap.atlas.utilities.threads.Pool;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.vividsolutions.jts.geom.Envelope;
+import com.vividsolutions.jts.geom.Polygon;
+import com.vividsolutions.jts.index.strtree.STRtree;
+
+/**
+ * Tests for {@link DynamicGridIndexBuilder}.
+ *
+ * @author mkalender
+ */
+public class DynamicGridIndexBuilderTest
+{
+    protected static final Logger logger = LoggerFactory
+            .getLogger(DynamicGridIndexBuilderTest.class);
+
+    @Test
+    public void testConsistencyOne()
```

Comment:
How about a private method for the common code that takes input of countries and boundary file, then we can expand consistency to any number of combinations. Otherwise the code is almost identical.